### ISSUE TITLE
Add new plan specific content

### ISSUE THREAD

**Issue by terotic** [City-of-Helsinki/cnh-ui#51]
- Organization URL (footer)
- Organization name (organization logo alt-text)
- Actions description (Front hero actions-button)
- Indicators description (Front hero indicators-button)

**Comment by terotic** [comment]
Blocked by: https://github.com/City-of-Helsinki/aplans/issues/6
